vendors/halo2/src/xor_shift_rng.rs

```diff
@@ -63,9 +67,9 @@ impl rand_core::RngCore for XORShiftRng {
 
 #[cfg(test)]
 mod test {
-    use rand_core::{RngCore, SeedableRng};
-
+    use super::RngState;
     use crate::consts::SEED;
+    use rand_core::{RngCore, SeedableRng};
```

Comment:
Did you intend to move this statement here?

Reply:
Oh no, this moved by `cargo fmt` automatically while saving. Is it okay to leave it like this ?

Reply:
Yup normally we separate formatting as a style commit, but it's minor, so just leave it!

Reply:
got it.